.happy/terraform/modules/batch/main.tf

```diff
@@ -0,0 +1,51 @@
+# This is a batch job
+#
+
+data aws_region current {}
+
+resource aws_batch_job_definition batch_job_def {
```

Comment:
is this a port of our existing batch pipelines or something else entirely?

Reply:
At the moment the batch-related code here is just a stub. I'm working with Andrey to get this fleshed out more fully.